package.json

```
@@ -1,76 +1,31 @@
 {
   "name": "sketch-api",
   "private": true,
-  "version": "2.0.0",
+  "version": "0.1.0",
   "author": "Sam Deane",
```

It's about time to change this

```suggestion
  "author": "Sketch",
```